frontend/fingertips-frontend/README.md

```diff
@@ -19,6 +19,14 @@ You can lint the code base (using [ESLint](https://eslint.org/)) by running:
 npm run lint
 ```
 
+### Prettier
```

Comment:
Updated the vscode settings to format on save for the project.

Reply:
Hrm, @steves-bjss and I discussed (in #9) about checking in IDE files and came down on the side of not doing it (as it's an open source repo and we want to be agnostic of IDEs used). Having said that, given the file's here I'm OK with adding the setting.

I still think it would be worth a suggestion to install the relevant extension for your IDE in the README though.

Reply:
Me and @williamwillis11 were discussing this. Do we want to add .vscode to .gitignore? since we were surprised that it was there. But liked that devs would get the same settings but that is a dev choice. 

I will update the readme though about the relevant extension.

Reply:
Yes, I think we should add it to the .gitignore.